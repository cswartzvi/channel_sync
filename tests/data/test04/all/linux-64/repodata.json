{
    "info": {
        "subdir": "linux-64"
    },
    "packages": {
        "a-1.0.tar.bz2": {
            "build": "",
            "build_number": 0,
            "depends": ["b >=1.0,<2.0"],
            "name": "a",
            "subdir": "linux-64",
            "version": "1.0"
        },
        "a-2.0.tar.bz2": {
            "build": "",
            "build_number": 0,
            "depends": ["b >=2.0,<3.0"],
            "name": "a",
            "version": "2.0"
        },
        "a-3.0.tar.bz2": {
            "build": "",
            "build_number": 0,
            "depends": ["b >=3.0"],
            "name": "a",
            "version": "3.0"
        },
        "b-1.0.tar.bz2": {
            "build": "",
            "build_number": 0,
            "depends": [ ],
            "name": "b",
            "subdir": "linux-64",
            "version": "1.0"
        },
        "b-2.0.tar.bz2": {
            "build": "",
            "build_number": 0,
            "depends": [ ],
            "name": "b",
            "version": "2.0"
        },
        "b-3.0.tar.bz2": {
            "build": "",
            "build_number": 0,
            "depends": [ ],
            "name": "b",
            "version": "3.0"
        }
    }
}