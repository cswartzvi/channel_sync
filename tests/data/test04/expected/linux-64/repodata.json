{
    "info": {
        "subdir": "linux-64"
    },
    "packages": {
        "a-2.0.tar.bz2": {
            "build": "",
            "build_number": 0,
            "depends": ["b >=2.0,<3.0"],
            "name": "a",
            "version": "2.0"
        },
        "a-3.0.tar.bz2": {
            "build": "",
            "build_number": 0,
            "depends": ["b >=3.0"],
            "name": "a",
            "version": "3.0"
        },
        "b-2.0.tar.bz2": {
            "build": "",
            "build_number": 0,
            "depends": [ ],
            "name": "b",
            "version": "2.0"
        },
        "b-3.0.tar.bz2": {
            "build": "",
            "build_number": 0,
            "depends": [ ],
            "name": "b",
            "version": "3.0"
        }
    }
}